static/app/components/group/seenInfo.tsx

```diff
@@ -19,13 +19,13 @@ type RelaxedDateType = React.ComponentProps<typeof TimeSince>['date'];
 type Props = {
   date: RelaxedDateType;
   dateGlobal: RelaxedDateType;
-  hasRelease: boolean;
   organization: Organization;
   projectId: string;
   projectSlug: string;
-  title: string;
+  className?: string;
```

Comment:
doesn't seem like classname is being used

Reply:
Good call, i forgot I figured it out without `styled(SeenInfo)` being used.